tox.ini

```diff
@@ -42,6 +42,6 @@ exclude = .ropeproject,.tox,sandbox
 show-source = True
 
 [pytest]
-norecursedirs = .tox .git .hg sandbox
+norecursedirs = .tox .git .hg sandbox build
```

Comment:
Ah, good fix. I always end up nuking the `build` folder when I run tox locally in a good while.